test/rubygems/test_gem_uri_parser.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'helper'
+require 'uri'
```

Comment:
I assume this require can be removed as well.

Reply:
For this one, I have required the `uri` module intentionally to test the correctness of the `Gem::UriParser.parse_uri`. Do you this test should be removed?

https://github.com/rubygems/rubygems/blob/4ceb2218dea88801ff98263ddada764139c2ac85/test/rubygems/test_gem_uri_parser.rb#L14-L16

Reply:
That makes sense. @deivid-rodriguez should we add `uri` at least to https://github.com/rubygems/rubygems/blob/master/dev_gems.rb to ensure it is present for tests?

Reply:
Yeah, it makes sense to me, at least to guarantee that we all use the same version of the `uri` gem for testing.

Reply:
Done at https://github.com/rubygems/rubygems/pull/4868/commits/77e3717e2d17d80d167b5ebec3985db8e23e4394.